portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,88 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  margin-top: 20px;
   width: 650px;
+  padding: 20px;
+  border: 5px solid #333;
+  border-radius: 6px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.body{
+    background:gainsboro;
+    color:black;
+    margin-left: auto;
+}
+
+.resume{
+    text-align: center;
+}
+
+h1{
+    text-align: center;
+    color:black
+}
+
+h2{
+    text-align: center;
+    color: cornflowerblue;
+}
+
+header{
+    text-align: center;
+    background: cornflowerblue;
+    padding-left: 20px;
+}
+
+.fa {
+  padding: 20px;
+  font-size: 30px;
+  width: 50px;
+  text-align: center;
+  text-decoration: none;
+  margin: 100px 4px;
+}
+ 
+.fa:hover {
+    opacity: 0.7;
+}
+ 
+.fa-linkedin {
```

Comment:
(non actionable comment) You've customized the font-awesome tags quite a bit! At some point, you might find that you don't need their stylesheet, since all of the styling is your own.